lib/mobility/plugins/backend.rb

```diff
@@ -45,7 +45,7 @@ def included(klass)
         klass.extend ClassMethods
 
         if backend
-          @backend_class = backend.build_subclass(klass, backend_options)
+          @backend_class = backend.build_subclass(klass, **backend_options)
```

Comment:
This adjustment was necessary to address the following error with Ruby 3.1 and 3.2.

```
Failures:

  1) Mobility::Plugins::Backend configuring defaults default without backend options calls setup_model on configured backend
     Failure/Error: @backend_class = backend.build_subclass(klass, backend_options)
     
       #<FooBackend (class)> received :build_subclass with unexpected arguments
         expected: (#<Class:0x0000000105782a48>, {:backend=>[:foo, {}]}) (keyword arguments)
              got: (#<Class:0x0000000105782a48>, {:backend=>[:foo, {}]}) (options hash)
     # ./lib/mobility/plugins/backend.rb:48:in `included'
     # ./spec/mobility/plugins/backend_spec.rb:325:in `include'
     # ./spec/mobility/plugins/backend_spec.rb:325:in `block (4 levels) in <top (required)>'
```